organizing_hub/views/call_views.py

```diff
@@ -86,6 +91,35 @@ class CallCampaignDetailView(LocalGroupPermissionRequiredMixin, DetailView):
     def get_local_group(self):
         return find_local_group_by_user(self.request.user)
 
+class CallCampaignUpdateView(
+    LocalGroupPermissionRequiredMixin,
+    SuccessMessageMixin,
+    UpdateView
+):
+    template_name = "calls/callcampaign_form.html"
+    form_class = CallCampaignUpdateForm
+    model = CallCampaign
+    permission_required = 'calls.change_callcampaign'
```

Comment:
Merge issue, this needs a feature flag now.

Reply:
👍